README.md

```diff
@@ -5,7 +5,22 @@
 Documentation for [iOS](https://developers.braintreepayments.com/start/hello-client/ios/v4)  
 Documentation for [Android](https://developers.braintreepayments.com/start/hello-client/android/v2)
 
+## TOCropViewController iOS
+TOCropViewController [GitHub](https://github.com/TimOliver/TOCropViewController)
```

Comment:
Mettre la description du projet : "A view controller that allows users to crop UIImage objects."